OsmAnd-java/src/main/java/net/osmand/search/core/SearchCoreFactory.java

```diff
@@ -601,10 +600,18 @@ public boolean search(final SearchPhrase phrase, final SearchResultMatcher resul
 
 			final BinaryMapIndexReader[] currentFile = new BinaryMapIndexReader[1];
 			Iterator<BinaryMapIndexReader> offlineIterator = phrase.getRadiusOfflineIndexes(BBOX_RADIUS,
-					SearchPhraseDataType.POI);
+					SearchPhraseDataType.POI); // WORLD obf(s) always + other obf(s) by radius
```

Comment:
comment not needed - code should be fixed

Reply:
comment was useful but removed ok